surveys/2022-wg-debugging-survey/questions.md

```diff
@@ -0,0 +1,361 @@
+# Survey questions
+One of the [compiler team's focuses for
+2022](https://blog.rust-lang.org/inside-rust/2022/02/22/compiler-team-ambitions-2022.html)
+is to improve the experience of debugging Rust code. We'd like to better
+understand your experiences using a debugger with Rust - what's been good,
+what's been bad and how your experience debugging Rust could be better.
+
+Unless you choose to enter your email, your answers will be anonymous. Any
+personal data you submit as a part of this survey will be handled in accordance
+with our policy:
+
+https://github.com/rust-lang/surveys/blob/main/documents/data-sharing-annual-community-survey.md
+
+We estimate it will take about ? minutes to complete.
+
+### Which operating systems do you use regularly for Rust debugging?
+Note: this is specifically about which systems you use for development and
+debugging not all the systems you target.
```

Comment:
This could be clearer - are you asking about both host and target systems, but only those target systems you debug? Or just host or just target...

Reply:
Reworded this